src/TerraformStartDirectoryPlugin.groovy

```diff
@@ -0,0 +1,42 @@
+import static TerraformValidateStage.ALL
+import static TerraformEnvironmentStage.ALL
+
+public class TerraformStartDirectoryPlugin implements TerraformValidateStagePlugin, TerraformEnvironmentStagePlugin, Resettable {
+    private static String directory = "./terraform/"
+
+    TerraformStartDirectoryPlugin() { }
```

Comment:
FYI: Once you define a class, Groovy will give you a default constructor method for free.  So while there's nothing wrong with defining it here, you're not required to provide it.